src/io/flutter/preview/PreviewView.java

```diff
@@ -186,12 +191,7 @@ public void initToolWindow(@NotNull ToolWindow toolWindow) {
 
     final DefaultActionGroup toolbarGroup = new DefaultActionGroup();
 
-    toolbarGroup.add(new AnAction(FlutterIcons.Center) {
-      @Override
-      public void actionPerformed(AnActionEvent e) {
-        Messages.showErrorDialog("Not implemented yet.", "TODO");
-      }
-    });
+    toolbarGroup.add(new QuickAssistAction(FlutterIcons.Center, "Surround with Center"));
```

Comment:
`Center Widget`? `Center`? `Warp with Center widget`?

Reply:
Nevermind, this is the name of the quick assist. Can you add an additional param for the action's tooltip?